### ISSUE TITLE
1.6 several recipes & statue drops broken

### ISSUE THREAD

**Issue by xGarionx** [scalda/Foolcraft_3#340]
## Issue Description
- Iron Gear not Craftable
- Emc Cell not craftable
- Statues dont drop 

## Reproduce Steps
- Progress in the game...

**Comment by Mrbysco** [comment]
What do you mean "Statues don't drop"

They got a 1% chance of dropping. And only drop when a real player kills them.

**Comment by xGarionx** [comment]
Yes i am aware of that ,after around ~2 hours of mindless grinding(with red matter aoe and a huge amount of mob spawn) and on a server with around ~20 people really actively playing its safe to assume the current config (or another tweak) made them undropable (since the config can be adjusted it should be sorted out with our next shedulded restart... but for general uses and you guys i thought i let you know)

**Comment by Revvilo** [comment]
I've had absolutely no troubles getting multiple statues with a middle of the road mob spawner and the Red Katar. So it's really odd you're having so much trouble.
Then again, maybe my idea of 'huge amount of mob spawns' is a bit crazier/more-game-crashing than yours.
Though, I can understand boosting the drop rates on a higher pop server, as natural spawns will be much fewer and I'd think they wouldn't allow a server crippling spawner for fast statue collecting.

**Comment by xGarionx** [comment]
I assure you ,you certainly dont have a crazier idea on huge amount... when i talk about huge amount i mean 30~50 mobs per red matter katar aoe hit.
However it was a config setting so thats sorted out... doesnt solve the issue with other recipes not crafting

**Comment by Soloun** [comment]
Is there anyway around the Iron Gear issue above? I can't craft an iron gear anymore.

**Comment by xGarionx** [comment]
right now only the 00-recipe file may help you to change that, maybe if you are not familiar with it and play on a server ask the community for mayb thier adjusted recipes :/ or to give you a gear to craft a compactor with an augment (at least that still works), saldy the augment needs an iron gear aswell

**Comment by Soloun** [comment]
Thanks mate, I'm playing single player and decided to just cheat in the gears and chuck some iron bars, a bit silly I guess but all good. I was hoping there might be another patch as I really like this pack but it is getting on a bit now I guess. Thanks for the reply.

**Comment by Soloun** [comment]
Hehe, I'm blonde I guess, when you got to craft the Iron Gear an Iron Fram is the result, HOWEVER a very small, on my UI, arrow also appears with a 2 next to it, if you click that Iron Gear is now the recipe result and you can craft the gear. I guess the defaults go mixed up or something? Either way it is craftable as long as you click that little arrow :)

**Comment by xGarionx** [comment]
Well we had several people on our server where the conflic recipe wasnt shown for that item in perticular ^^ soo glad it worked out then
